bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -155,6 +191,10 @@
 		<label>Intended Boiler Temperature</label>
 		<description>Gets the intended temperature of this boiler</description>
 		<category>heating</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
```suggestion
			<tag>Calculation</tag>
```

Reply:
Umm. I think this is probably an example of the (in)famous read-only `SetPoint` ..

Reply:
.. so changing to `Setpoint.Temperature`